components/text_input/README.md

```diff
@@ -103,12 +99,12 @@ This library is updated on a regular basis; by subscribing to the design library
 ### CSS
 ```
 form {
-  font-family: 'PT Sans','Calibri', 'Arial', 'Sans Serif';
+  font-family: 'Noto Sans','Calibri', 'Arial', 'Sans Serif';;
```

Comment:
i don't think font family is necessary here. Ideally we'd have that set in a top level element such as the body

Reply:
that or provide a comment explaining that we are using font family here for the example display purposes, but ideally you'd be setting this up in a higher page element.

Reply:
It seemed to not affect the text within the input field but I could be wrong